src/elements/content-sidebar/activity-feed/SelectableActivityCard.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../common/variables';
+
+@keyframes bcs-SelectableActivityCard-hover {
+    from {
+        background-color: transparent;
+    }
+
+    to {
+        background-color: $bdl-box-blue-05;
+    }
+}
+
+.bcs-SelectableActivityCard {
+    &[aria-disabled='false'] {
+        cursor: pointer;
+
+        &:hover,
+        &:focus {
+            outline: none;
+            box-shadow: 0 0 8px 0 rgba(114, 132, 142, .16);
+            animation: bcs-SelectableActivityCard-hover 1s ease-in-out 25ms 1 normal forwards;
```

Comment:
Is the one second duration a little long for a hover effect? Should this be a transition, since it only plays once? If it needs to be an animation, it looks like some of these properties are their default values and can be omitted.

Reply:
Good point, I'll adjust this